### PR TITLE
docs: Quick Start v2.0

### PR DESCRIPTION
**Pull request by tstromberg**
Here's the new quick start guide!

* Unifies all 3 platforms onto a single page
* Minimizes contextual overload
* Now mentions tunnel, multi-cluster

Preview URL: https://deploy-preview-7563--kubernetes-sigs-minikube.netlify.com/docs/start/

### CODE REVIEW

File: site/assets/scss/_variables_project.scss [hunk 1]
```diff
@@ -3,16 +3,17 @@
...
+$mk-midlight: #e8f3f3;
 $mk-light:#C7EAEC;
 $mk-verydark: darken($mk-dark, 15%);
 
 // bootstrap colors
 $dark: #403F4C !default;
 $blue: #72A1E5 !default;
 $orange: #BA5A31 !default;
-$gray-100: #f8f9fa !default;
+$gray-100: #f6f6f8 !default;
```
Comment (medyagh) [inline site/assets/scss/_variables_project.scss:14]: the new gray is a bit too dark, maybe just little bit shad of lighter (I am reffering to the background of the tabs)  not sure which one of the grays is used there

File: site/assets/scss/_variables_project.scss [hunk 2]
```diff
@@ -31,7 +32,30 @@ $danger: #ED6A5A !default;
...
+}
+
+
+h1,h2,h3,h4,h5 {
+    font-family: 'Open Sans', sans-serif !important;
+}
+
+.step {
+    margin-top: 2em;
+    font-family: 'Open Sans', sans-serif !important;
```
Comment (medyagh) [inline site/assets/scss/_variables_project.scss:48]: not sure which one is the font for the Code Blocks
but how about we make the code block fonts, a Code Font...Like Currier or whatever looks Computer-Font

**Comment by medyagh** [comment]
thanks for doing it, it is so much better with less Clicks u get to Linux or Windows Quick Start guide

File: site/content/en/docs/start/_index.md [hunk 3]
```diff
@@ -1,8 +1,190 @@
...
+```shell
+choco install minikube
+```
+
+Otherwise, download and run the [Windows installer](https://storage.googleapis.com/minikube/releases/latest/minikube-installer.exe)
+
+{{% /tab %}}
+{{% /tabs %}}
+
+<h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">2</strong></span>Start your cluster</h2>
```
Comment (medyagh) [inline site/content/en/docs/start/_index.md:79]: Thanks for adding that beautiful Gui Step Number, I look forward to see follow up PRs to make all the Documentation look like this for Numbered items . for example Push image docs.
(we should make an issue for someone to pick up this task) this Graphical Number Icon adds a lot of usability to for ppl who dont like to read docs

File: site/assets/scss/_variables_project.scss (hunk 2 above)
Comment (tstromberg) [reply site/assets/scss/_variables_project.scss:48]: I'm a bit confused - the code blocks are a code font right now: "Inconsolata". It seemed to be the one that worked the best with the other fonts (Lora & Open Sans), and decently narrow. Not your taste?

File: site/assets/scss/_variables_project.scss (hunk 1 above)
Comment (tstromberg) [reply site/assets/scss/_variables_project.scss:15]: I made it a lighter form of minikube-blue again. How's it look?

File: site/assets/scss/_variables_project.scss (hunk 2 above)
Comment (tstromberg) [reply site/assets/scss/_variables_project.scss:49]: Increased font size just a hair for clarity.

File: site/content/en/docs/start/_index.md (hunk 3 above)
Comment (tstromberg) [reply site/content/en/docs/start/_index.md:79]: Thanks!

**Pull request merged**